server/game/GameActions/ReadyAction.js

```diff
@@ -8,6 +8,9 @@ class ReadyAction extends CardGameAction {
     }
 
     canAffect(card, context) {
+        if (card.game.currentPhase === 'ready' && !card.readiesDuringReadyPhase()) {
```

Comment:
I think `doesNotReady` is actually not used anywhere. Maybe Kiri Giltspine could have used it in the past.
I think for consistency, we could follow how Kiri Giltspine is done or Storm Surge.

Reply:
I did look at Kiri Giltspine but it is actually worded slightly differently because it is a lasting effect tied to the controller at the time the effect is put into place.  So if it changes sides (like in one of its tests), it actually _would_ ready.  But Frost Giant is a persistent effect and does not matter who controls it, so i can't just use the same code there.  Let me know if I'm missing something.

I updated Storm Surge to use `doesNotReady` and the tests still pass, and it looks a bit cleaner to me.  What do you think?

Reply:
Can you commit the  updated Storm Surge?

Reply:
Oops, I thought I had, sorry.  It's there now, ready for another look.